controllers/storageclaim_controller.go

```diff
@@ -286,15 +286,15 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 		var storageClaimStorageType providerclient.StorageType
 		switch r.storageClaim.Spec.Type {
 		case "block":
-			storageClaimStorageType = providerclient.StorageTypeBlockpool
+			storageClaimStorageType = providerclient.StorageTypeBlock
 		case "sharedfile":
-			storageClaimStorageType = providerclient.StorageTypeSharedfilesystem
+			storageClaimStorageType = providerclient.StorageTypeSharedfile
```

Comment:
```suggestion
			storageClaimStorageType = providerclient.StorageTypeSharedFile
```

Reply:
will be taken up in a different PR to cut across all the components and change to `SharedFile` and `Block` as the values.